crates/cubecl-core/src/compute/kernel.rs

```diff
@@ -7,6 +7,7 @@ use cubecl_runtime::ExecutionMode;
 /// A kernel, compiled in the target language
 pub struct CompiledKernel<C: Compiler> {
     pub name: Option<&'static str>,
+    pub kernel_name: String,
```

Comment:
What is the difference between `name` and `kernel_name` ?

Reply:
Sure. I added that back, but changed the names to "entrypoint_name" (the new one) and "debug_name" (the old one).

To make it less confusing I added some docs:

![image](https://github.com/user-attachments/assets/1c3e7d86-6b3a-454d-bbaa-1caa8118a48b)